events/wc_draenei_events.txt

```diff
@@ -0,0 +1,277 @@
+namespace = WCDRN
+
+############################
+#	Crashlanding of Exodar
+############################
+
+# Creates the Draenei and characters
+character_event = {
+	id = WCDRN.99
+
+	trigger = {
+		NOT = { has_global_flag = exodar_has_crashed_flag }
+		year = 610
+	}
+	
+	immediate = {
+		create_character = {
+			name = Velen
+			dynasty = 140000
+			culture = draenei 
+			religion = naaru
+			attributes = {
+				martial=5 diplomacy=6 stewardship=6 intrigue=3 learning=7
+			}
+			trait=mastermind_theologian
+			trait=creature_draenei
+			trait=class_priest_9
+			trait=honest trait=patient trait=just trait=zealous trait=kind trait=sorcerer trait=immortal
+			age = 600
+		}
+		new_character = {
+			3320 = {
+				capital_holding = { convert_tribal_to_temple_effect = yes }
+				county = { grant_title_no_opinion = { target = PREVPREV } }
+				culture = PREV
+				religion = PREV
+			}
+			3322 = {
```

Comment:
What if it's player province?

Reply:
If it's player province, a preventive measure should be implemented to avoid getting instant game over due to becoming unlanded. The general layout was chosen to reflect the major operation sites (sans Blood Watch). Possible options include vassalization offer if a minor or war.

Reply:
Done.